src/status_im2/contexts/share/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.share.events
+  (:require [utils.re-frame :as rf]
+            [status-im2.common.toasts.events :as toasts]
+            [quo2.foundations.colors :as colors]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.react :as react]))
+
+(rf/defn open-profile-share-view
+  {:events [:share/open]}
+  [{:keys [db]}]
+  {:dispatch [:open-modal :profile-share {}]})
```

Comment:
:dispatch shouldn't be used in event handles, `:share/open` looks redundant, why don't use `:open-modal` directly ?

Reply:
above code is redundant, I'll get rid of this.